### PR TITLE
Normalize Element Content

### PR DESCRIPTION
**Pull request by NikolasKomonen**
```
  <a>
    aa
      aa

    aa
  </a>
```

Will stay the same after formatting if wanted

### CODE REVIEW

File: org.eclipse.lsp4xml/src/main/java/org/eclipse/lsp4xml/settings/XMLFormattingOptions.java [hunk 1]
```diff
@@ -89,8 +91,21 @@ public boolean isJoinCommentLines() {
 		}
 	}
 
-	public void setJoinCommentLines(final boolean joinCommentLines) {
-		this.putBoolean(XMLFormattingOptions.JOIN_COMMENT_LINES, Boolean.valueOf(joinCommentLines));
+	public void setJoinCommentLines(final boolean normalizeElementContent) {
```
Comment (fbricon) [inline org.eclipse.lsp4xml/src/main/java/org/eclipse/lsp4xml/settings/XMLFormattingOptions.java]: nope

File: org.eclipse.lsp4xml/src/main/java/org/eclipse/lsp4xml/settings/XMLFormattingOptions.java [hunk 2]
```diff
@@ -89,8 +91,21 @@ public boolean isJoinCommentLines() {
...
+		this.putBoolean(XMLFormattingOptions.JOIN_COMMENT_LINES, Boolean.valueOf(normalizeElementContent));
+	}
+
+	public boolean isNormalizeElementContent() {
```
Comment (fbricon) [inline org.eclipse.lsp4xml/src/main/java/org/eclipse/lsp4xml/settings/XMLFormattingOptions.java]: isJoinContentLines to be consistent with isJoinCommentLines

File: org.eclipse.lsp4xml/src/main/java/org/eclipse/lsp4xml/settings/XMLFormattingOptions.java [hunk 3]
```diff
@@ -15,6 +15,7 @@
 	private static final String JOIN_CDATA_LINES = "joinCDATALines";
 	private static final String FORMAT_COMMENTS = "formatComments";
 	private static final String JOIN_COMMENT_LINES = "joinCommentLines";
+	private static final String NORMALIZE_ELEMENT_CONTENT = "normalizeElementContent";
```
Comment (fbricon) [inline org.eclipse.lsp4xml/src/main/java/org/eclipse/lsp4xml/settings/XMLFormattingOptions.java]: JOIN_CONTENT_LINES = "joinContentLines"

**Comment by angelozerr** [comment]
Thanks @NikolasKomonen ! Please update wiki at https://github.com/angelozerr/lsp4xml/wiki/Configuration#formatting

**Comment by angelozerr** [comment]
Ooops sorry @fbricon. Do you want I revert this commit?

**Pull request merged**